src/renderer/component/fileCard/view.jsx

```diff
@@ -89,25 +89,37 @@ class FileCard extends React.PureComponent {
             className="card__link"
           >
             <CardMedia title={title} thumbnail={thumbnail} />
-            <div className="card__title-identity">
+          </Link>
```

Comment:
Instead of closing the link here, does it work if you keep it at the bottom and encapsulate the new link? You should still be able to click on the file if your mouse is below the channel name.

Hope that makes sense

Reply:
That was my first thought too. In that case it takes you to the file page and not the channel page. Though I think it actually takes you to the channel page and then the file page (calling the navigate function twice?) because when I click the channel link and then click Back it does take me to the channel page.

Reply:
I ended up wrapping links around the image, title, and price to avoid this.

Reply:
You should be able to use `e.stopPropagation` to fix that. We still want to keep just one link for the whole card. You could update the `<Link />` component to take a prop `noPropagation` which would call `e.stopPropagation()` before doing the navigate.

I think that would work.